main/urls.py

```diff
@@ -100,3 +101,7 @@
     urlpatterns += [
         path("__debug__/", include(debug_toolbar.urls)),
     ]
+if features.is_enabled(features.ENABLE_NEW_DESIGN):
```

Comment:
This doesn't seem to work. If I disable the flag (using my Posthog account) I can still get to the catalog page. Not sure what's going on there. 

In testing, though, I think there's another problem here. The URLconf is only (apparently) parsed once per worker so if we toggled the flag and then reloaded, it wouldn't prevent access here. (I don't see an event coming through on page load, in other words - just when I bounce the web container.) So, this should probably be in the view, which will get evaluated on page load.